pkgs/nbxplorer/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, bash, dotnet, callPackage, writeText, makeWrapper,
```

Comment:
I realize I have extra arguments here I forgot to remove after I refactored everything. Will remove these